apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/RestParameters.java

```diff
@@ -0,0 +1,100 @@
+
+    public static final String PASSPHRASE_PARAM_NAME="passphrase";
+    public static final String SECRET_PHRASE_PARAM_NAME ="secretPhrase";
+    public static final String CODE2FA_PARAM_NAME="code2FA";
+
+    public static Map<String, String> parseRequestParameters(ContainerRequestContext requestContext, String ... params){
+        Map<String, String> parsedParams = new HashMap<>();
+        MultivaluedMap<String, String> requestParams;
+        requestParams = ((PostMatchContainerRequestContext) requestContext).getHttpRequest().getDecodedFormParameters();
+        requestParams.putAll(requestContext.getUriInfo().getQueryParameters(true));
+
+        Arrays.stream(params).forEach(p -> parsedParams.put(p, requestParams.getFirst(p)));
+
+        return parsedParams;
+    }
+
+    public static Integer parseHeight(String heightValue, int maxHeight) throws RestParameterException {
+        if (heightValue == null){
+            return -1;
+        }
+        Integer height = parseInt(heightValue, 0, maxHeight, "height");
+        return height;
+    }
+
+    public static Integer parseInt(String intStrValue, int min, int max, String paramName) throws RestParameterException {
+        if (intStrValue != null) {
+            try {
+                int intValue = Integer.parseInt(intStrValue);
+                return parseInt(intValue, min, max, paramName);
+            } catch (NumberFormatException e) {
+                throw new RestParameterException(ApiErrors.INCORRECT_VALUE, paramName, intStrValue);
+            }
+        }
+        return min;
+    }
+
+    public static Integer parseInt(Integer intValue, int min, int max, String paramName) throws RestParameterException {
+        if (intValue != null) {
+            if (intValue < min || intValue > max) {
+                throw new RestParameterException(ApiErrors.INCORRECT_VALUE, paramName, intValue);
+            }
+            return intValue;
+        }
+        return min;
+    }
+
+    public static int parseInt(String paramValue, String paramName, int min, int max, boolean isMandatory) throws RestParameterException {
+        if (paramValue == null || paramValue.isEmpty()) {
+            if (isMandatory) {
+                throw new RestParameterException(ApiErrors.MISSING_PARAM, paramName);
+            }
+//            return 0;
+            return min; // let's return minimal permitted
+        }
+        try {
+            int value = Integer.parseInt(paramValue);
+            if (value < min || value > max) {
+                throw new RestParameterException(ApiErrors.OUT_OF_RANGE_NAME_VALUE, paramName, value, min, max);
+            }
+            return value;
+        } catch (RuntimeException e) {
+            throw new RestParameterException(ApiErrors.INCORRECT_VALUE, paramName, paramValue);
+        }
+    }
+
+    public static long parseAccountId(String account) throws RestParameterException {
```

Comment:
ParameterParser implement same logic, it there a reason to have 2 similar classes/methods?